services/slam/builtin/builtin.go

```diff
@@ -985,21 +974,10 @@ func (slamSvc *builtIn) getAndSaveDataSparse(
 			return nil, err
 		}
 		for i, filename := range filenames {
-			//nolint:gosec
-			f, err := os.Create(filename)
+			err = dataprocess.WriteBytesToFile(images[i], filename)
 			if err != nil {
 				return filenames, err
 			}
```

Comment:
[nit] 
```suggestion
			if err := dataprocess.WriteBytesToFile(images[i], filename); err != nil {
				return filenames, err
			}
```

Reply:
Ah, I had forgotten about this syntax trick! Thanks